test/metabase/query_processor_test/date_bucketing_test.clj

```diff
@@ -900,7 +900,7 @@
   [^TimestampDatasetDef this]
   (let [interval-seconds (.intervalSeconds this)]
     (mt/dataset-definition
-     (str "checkins_interval_" interval-seconds)
+     (str "interval_" interval-seconds)
```

Comment:
this seemed like a good enough name for the dataset, since there's only one table anyway: "checkins"